src/main/java/app/linguistai/bmvp/model/stats/UserLoggedDate.java

```diff
@@ -0,0 +1,31 @@
+package app.linguistai.bmvp.model.stats;
+
+import app.linguistai.bmvp.model.User;
+import app.linguistai.bmvp.model.embedded.UserLoggedDateId;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.sql.Date;
+
+@Data
+@Entity
+@Table(name = "user_logged_date")
+@NoArgsConstructor
+@AllArgsConstructor
+@IdClass(UserLoggedDateId.class)
+public class UserLoggedDate {
+
+	@Id
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "user_id", referencedColumnName = "id", nullable = false)
+	@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+	private User user;
```

Comment:
please add `@NotNull` annotation to this field (this makes this field not nullable for the Java code, as `nullable = false` is for the DB)